code/datums/traits/negative.dm

```diff
@@ -575,6 +575,8 @@
 	if (!reagent_type)
 		reagent_type = pick(drug_list)
 	reagent_instance = new reagent_type()
+	if(!H.reagents.addiction_list)
+		H.reagents.addiction_list = list()
 	H.reagents.addiction_list.Add(reagent_instance)
```

Comment:
```suggestion
	LAZYADD(H.reagents.addiction_list, reagent_instance)
```

...and then you'll be able to remove the above 2 lines.

Reply:
Ah, thanks!